cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java

```diff
@@ -170,15 +170,16 @@ public AssignmentDTO addAssignment(@RequestBody AssignmentDTO assignmentDTO) {
 			Assignment assignment = new Assignment();
 			assignment.setName(assignmentDTO.assignmentName);
 				
-			SimpleDateFormat formatter = new SimpleDateFormat("dd-MMM-yyyy", Locale.ENGLISH);
+			SimpleDateFormat formatter = new SimpleDateFormat("dd-MM-yyyy", Locale.ENGLISH);
 			Date dueDate = null;
 			try {
 				dueDate = (Date) formatter.parse(assignmentDTO.dueDate);
 			} catch (ParseException e) {
 				// TODO Auto-generated catch block
 				e.printStackTrace();
 			}	
-			assignment.setDueDate(dueDate);
+			
+			assignment.setDueDate(new java.sql.Date(dueDate.getTime()));
```

Comment:
We wanted a `java.sql.Date` and not a `java.util.Data`? Only asking because I see the import above was updated.

Reply:
Yes, because in Assignment.java,  dueDate is a java.sql.Date.